modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -48,7 +49,9 @@ public static List<Map<String, Object>> getCodeEditorElements(
 					_objectFieldLocalService.getObjectFields(
 						objectDefinitionId),
 					objectField -> HashMapBuilder.put(
-						"content", objectField.getName()
+						"content",
+						StringUtil.removeSubstring(
+							objectField.getDBColumnName(), StringPool.UNDERLINE)
```

Comment:
Hi @seltong,

Slack me tomorrow to talk about this approach, I believe we can try to keep the objectFieldName for Custom Objects and get the DBColumnName for System ones.

Thanks!